src/libsync/capabilities.h

```diff
@@ -97,6 +97,31 @@ struct OWNCLOUDSYNC_EXPORT SpaceSupport
 class OWNCLOUDSYNC_EXPORT Capabilities
 {
 public:
+    struct OWNCLOUDSYNC_EXPORT AppProviders
+    {
+        /**
+         "app_providers": [
+                {
```

Comment:
The indentation is a bit off.

What's the purpose of this comment? Documenting the format? This could be done within the `.cpp` file as well. Right now, however, it is a docstring for the constructor.